examples/login.rs

```diff
@@ -0,0 +1,14 @@
+use rustforce::{Client, Error};
+// use std::env;
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+  let mut client = Client::new("".to_string(), "".to_string());
+  client
+    .login_by_soap(
+      "team@xe432.elastify.eu".to_string(),
+      "hchkyMTv2f2qmt2euEVqX9dxG2RvLaeMdRveXP2Cb3kBWKDsliCkVhk3RvTeQZDPnCmhyHu".to_string(),
```

Comment:
Instead of hardcoded credentials, would you like to replace to env::vars?

https://github.com/tzmfreedom/rustforce/blob/150e325c2cac5956738b03d9d62a2dd31ecacf1f/examples/create.rs#L26-L27

Reply:
Sorry about that I was testing it with some credentials I had. I will fix that right away.